lib/browser/api/menu.js

```diff
@@ -164,7 +164,8 @@ Menu.buildFromTemplate = function (template) {
   const sorted = sortTemplate(filtered)
 
   sorted.forEach((item) => {
-    if (typeof item !== 'object') {
+    if ((item === null) || (typeof item !== 'object') ||
```

Comment:
Can you please move this condition to a separate function? It looks complicated.